packages/dashboard/src/API.tsx

```diff
@@ -22,6 +35,12 @@ export interface PluginServices {
             derivationPath?: string,
             initialPassword?: string | undefined,
         ): Promise<string>
+        generateAddressFromMnemonic(
+            name: string,
+            mnemonic: string,
+            derivationPath?: string,
+            initialPassword?: string | undefined,
```

Comment:
Align with implementation.

```suggestion
            initialPassword?: string,
```